docs/manage/security/tls.rst

```diff
@@ -9,9 +9,13 @@ data being transmitted from being modified or read while it is in transit and al
 verify the identity of the server (in this case, the Determined master). Determined can be
 configured to use TLS for all connections made to the master. That means that all CLI and WebUI
 connections will be secured by TLS, as well as connections from agents and tasks to the master.
+Proxied connections between the master and :ref:`notebooks <notebooks>` will always use TLS by
+default. Proxied connections between the master and :ref:`shells <shells>` will always use the SSH
+by default.
+
```

Comment:
By default, proxied connections between the master and :ref:`notebooks <notebooks>` use TLS, while those with :ref:shells <shells> use SSH.